tasks/README.md

```diff
@@ -173,13 +173,19 @@ by the webhook which automatically starts a release runner.
 
 ### Automated testing
 
-When a pull request event is received, the webhook will scan the pull request
-for tests that need to be run. These will be put in a distributed queue to later
-be consumed by other task bots. If "[no-test]" is in the title of the pull
+When a pull request event is received, the webhook will publish the event to a
+distributed queue. These events are then consumed and interpreted by task bots
+who will publish tasks to the relevant queues, to be consumed again by task bots
+at a later point in time. If "[no-test]" is in the title of the pull
 request, it won't be scanned (this is implemented in cockpit's
 bots/tests-scan).
```

Comment:
Let's drop the [no-test] part here. With the recent changes, cockpituous is now completely oblivious of this.

Reply:
ack